rnndb/graph/g80_texture.xml

```diff
@@ -6,157 +6,312 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
+			<doc>Tells texture unit to smear border color from R to RGBA channels</doc>
+		</bitfield>
+		<bitfield pos="31" name="USE_TEXTURE_HEADER_V2" type="boolean">
+			<doc>if 0, then byte 7 of the TIC is used to store the color key value. Should always be 1.</doc>
+		</bitfield>
 	</reg32>
 
 	<reg32 offset="0x14" name="5">
-		<bitfield high="31" low="28" name="LAST_LEVEL" />
+		<bitfield high="31" low="28" name="MAP_MIP_LEVEL" />
 		<bitfield high="27" low="16" name="DEPTH" />
 		<bitfield high="15" low="0" name="HEIGHT">
 			<doc>Set to 0 for height of 65536.</doc>
 		</bitfield>
 	</reg32>
 
+	<reg32 offset="0x18" name="6">
+		<bitfield high="4" low="0" name="TRILIN_OPT">
+			<doc>slope for trilinear optimization, e2m3, controlled by USE_HEADER_OPT_CONTROL</doc>
+		</bitfield>
+		<bitfield high="17" low="5" name="MIP_LOD_BIAS">
+			<doc>bias added to lod, s5.8 number, use controlled by USE_HEADER_OPT_CONTROL</doc>
+		</bitfield>
+		<bitfield high="22" low="19" name="ANISO_BIAS">
+			<doc>aniso ratio (minor/major) is multiplied by 1+anisoBias to fatten footprints, u0.4. Spread functions modify the spread as a function of distance = fine? lodf: 1-lodf. Note that sample count is adjusted accordingly to cover the entire footprint</doc>
+		</bitfield>
+		<bitfield high="24" low="23" name="ANISO_FINE_SPREAD_FUNC" type="g80_aniso_spread_func">
+			<doc>aniso optimization function used for fine level (nv3x &amp; nv4x use 2)</doc>
+		</bitfield>
+		<bitfield high="26" low="25" name="ANISO_COARSE_SPREAD_FUNC" type="g80_aniso_spread_func">
+			<doc>aniso optimization function used for coarse level (nv3x &amp; nv4x use 0)</doc>
+		</bitfield>
+		<bitfield high="29" low="27" name="MAX_ANISOTROPY" type="g80_max_anisotropy">
+			<doc>max aniso ratio (major/minor), use controlled by USE_HEADER_OPT_CONTROL</doc>
+		</bitfield>
+		<bitfield high="31" low="30" name="ANISO_FINE_SPREAD_MODIFIER" type="g80_aniso_spread_modifier">
+			<doc>spread modifier modifies fine spread only, not sample count</doc>
+		</bitfield>
+	</reg32>
+
 	<reg32 offset="0x1c" name="7">
-		<bitfield high="3" low="0" name="BASE_LEVEL" />
-		<bitfield high="7" low="4" name="MAX_LEVEL" />
-		<bitfield high="15" low="12" name="MS_MODE" type="g80_msaa_mode">
-			<doc>Like g80/gf100_msaa_mode, dimensions are unaffected.</doc>
+		<bitfield high="3" low="0" name="RES_VIEW_MIN_MIP_LEVEL">
+			<doc>min LOD for texture resource views</doc>
+		</bitfield>
+		<bitfield high="7" low="4" name="RES_VIEW_MAX_MIP_LEVEL">
+			<doc>max LOD for texture resource views</doc>
+		</bitfield>
+		<bitfield pos="8" name="HEIGHT_MSB">
+			<doc>bit 16 of the texture's height, bits 0-15 are in Dword5</doc>
+		</bitfield>
+		<bitfield high="15" low="12" name="MULTI_SAMPLE_COUNT" type="g80_msaa_mode">
+			<doc>Like g80/gf100_msaa_mode, dimensions are unaffected. return value for the TEX_HEADER_TEXTURE_TYPE ISA instruction</doc>
+		</bitfield>
+		<bitfield high="27" low="16" name="MIN_LOD_CLAMP">
```

Comment:
type="ufixed" radix="8"

Please do this in other places where you have fixed point bitfields (use type="fixed" for signed)